frontend/src/metabase/components/InputBlurChange/InputBlurChange.tsx

```diff
@@ -58,7 +58,7 @@ const InputBlurChange = (props: InputBlurChangeProps) => {
     [normalize, onBlurChange, value],
   );
 
-  useUnmount(() => {
+  useUnmountLayout(() => {
```

Comment:
React 17 now runs useEffect cleanup functions asynchronously, so inputRef was coming back as undefined. Leveraging useLayoutEffect's clean up function allows to access the input right before it's unmounted.